plugins/builds/index.js

```diff
@@ -27,30 +27,46 @@ const workflowParser = require('screwdriver-workflow-parser');
  * @param  {Boolean}  [config.start]        Whether to start the build or not
  * @return {Promise}
  */
-function createBuild({ jobFactory, buildFactory, eventFactory, pipelineId, jobName, username,
-    scmContext, build, start }) {
-    return Promise.all([
-        eventFactory.get(build.eventId),
-        jobFactory.get({
-            name: jobName,
-            pipelineId
-        })
-    ]).then(([event, job]) => {
-        if (job.state === 'ENABLED') {
-            return buildFactory.create({
-                jobId: job.id,
-                sha: build.sha,
-                parentBuildId: build.id,
-                eventId: build.eventId,
-                username,
-                configPipelineSha: event.configPipelineSha,
-                scmContext,
-                start: start !== false
-            });
-        }
-
-        return null;
+async function createBuild({ jobFactory, buildFactory, eventFactory, pipelineFactory,
+    pipelineId, jobName, username, scmContext, build, start }) {
+    const event = await eventFactory.get(build.eventId);
```

Comment:
Do we need some error handling for these `await` statements?

Reply:
May be error will be catched here?
`createBuild` method is called from `triggerNextJobs`, and `triggerNextJobs` is called here.
https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/builds/update.js#L208